src/components/MainBanner/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { IncrementPercentage } from '../IncrementPercentage'
+import { DataCard } from  '../DataCard'
+
+import './MainBanner.scss'
+import mexicoImage from '../../assets/mexico.jpg'
+
+export const MainBanner = () => {
+	const [country, setCountry] = useState([])
+	const API = 'https://api.unsplash.com/search/photos?client_id=rTLPnTqzq-rK3TnatKGFvmVeFGaZeYToaKUyzwcJOiA&query='
+	const exampleCountry = 'mexico'
+
+	useEffect(() => {
+		window.fetch(`${API}${exampleCountry}`)
+			.then(res => res.json())
+			.then(response => {
+				setCountry(response)
+			})
+	}, [])
+
+	useEffect(() => {
+		const countryImage = document.querySelector('.country-image')
+		countryImage.style.backgroundImage = `url(${mexicoImage})`
+	})
+
+	return (
+		<React.Fragment>
+			<h2>Overview</h2>
+			<section className="main-banner">
+				<div className="country-image"></div>
+				<ul className="menu">
+					<li>
+						<Link to="/" className="menu__link menu__link--active">Total</Link>
```

Comment:
why a Link ??? what is the flow from that information?

Reply:
I'm not sure how this menu will work so I used Link just to make the layout, but now that you mention it I think it's better to use buttons, don't you think?

Reply:
I'm not sure about the interaction, for me, use a span it's ok if will not have a interactions